contracts/extensions/ERC721APausable.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+// Creator: Chiru Labs
+
+pragma solidity ^0.8.4;
+
+import '../ERC721A.sol';
+import "@openzeppelin/contracts/security/Pausable.sol";
+
+error ContractPaused();
+
+/**
+ * @dev ERC721A token with pausable token transfers, minting and burning.
+ *
+ * Based off of OpenZeppelin's ERC721Pausable extension.
+ *
+ * Useful for scenarios such as preventing trades until the end of an evaluation
+ * period, or having an emergency switch for freezing all token transfers in the
+ * event of a large bug.
+ */
+abstract contract ERC721APausable is ERC721A, Pausable {
+    /**
+     * @dev See {ERC721A-_beforeTokenTransfers}.
+     *
+     * Requirements:
+     *
+     * - the contract must not be paused.
+     */
+    function _beforeTokenTransfers(
+          address from,
+          address to,
+          uint256 startTokenId,
+          uint256 quantity
+      ) internal override virtual {
```

Comment:
Fix indentation to 4 spaces.

Reply:
Fixed! Just ran the linter. Looks like we hadn't ran it in a bit.